api/swagger.yaml

```diff
@@ -1923,6 +1923,38 @@ definitions:
                 type: "array"
                 items:
                   type: "string"
+          Secrets:
+            description: "Secrets contains references to zero or more secrets that will be exposed to the container."
```

Comment:
I think probably either service or the container is fine. But yes service will make it clear that this is only available in swarm mode. The PR has been updated. Thanks.